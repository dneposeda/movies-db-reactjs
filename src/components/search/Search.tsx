import React from "react";

const Search = () => {
    const handleClick = (e) => {
        e.preventDefault();
        console.log('Кликнули кнопке поиска.');
    }

    return (
        <div className="search">
            <form className="form-group" action="">
                <input data-search="searchFilms" type="text" className="form-control search__input" placeholder="Search"/>
                <button data-search="searchFilmsSubmit" type="submit" className="btn btn-lg search__btn" onClick={handleClick} >Search</button>
            </form>
        </div>
    )
}

export default Search;
