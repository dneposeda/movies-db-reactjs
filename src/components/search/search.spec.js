import React from 'react';
import Search from './Search';

describe('Search', () => {
    it('renders correctly', () => {
        const component = shallow (
            <Search />
        );
        expect(component).toMatchSnapshot();
    });
});
