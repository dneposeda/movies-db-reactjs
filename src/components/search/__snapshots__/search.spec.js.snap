// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div
  className="search"
>
  <form
    action=""
    className="form-group"
  >
    <input
      className="form-control search__input"
      data-search="searchFilms"
      placeholder="Search"
      type="text"
    />
    <button
      className="btn btn-lg search__btn"
      onClick={[Function]}
      type="submit"
    >
      Search
    </button>
  </form>
</div>
`;
